mitiq/interface/conversions.py

```diff
@@ -259,27 +259,28 @@ def qprogram_modifier(
 
 
 def accept_qprogram_and_validate(
-    cirq_circuit_modifier: Callable[..., Any]
+    cirq_circuit_modifier: Callable[..., Any],
+    one_to_many: Optional[bool] = False,
 ) -> Callable[..., Any]:
     """This decorator performs two functions:
 
-        1. Transforms a function of signature (cirq.Circuit -> Any) to
-        (QPROGRAM -> Any)
+        1. Transforms a function of signature (cirq.Circuit -> cirq.Circuit)
+        to (QPROGRAM -> QPROGRAM).
```

Comment:
Prior to this change, this docstring was more general than what was actually supported. Is that right? I.e. this isn't because a change you made in the code, but rather bringing the docstring in line with the existing functionality.

Reply:
Exactly!